apps/spruce/src/analytics/waterfall/useWaterfallAnalytics.ts

```diff
@@ -23,7 +23,8 @@ type Action =
   | { name: "Changed page"; direction: "next" | "previous" }
   | { name: "Deleted one filter chip" }
   | { name: "Deleted all filter chips" }
-  | { name: "Toggled task icon legend"; open: boolean };
+  | { name: "Toggled task icon legend"; open: boolean }
+  | { name: "Viewed waterfall beta modal"; "beta.enabled": boolean };
```

Comment:
this is just a suggestion; `beta_features` could be a namespace that we reuse later for other beta features? 
```suggestion
  | { name: "Viewed waterfall beta modal"; "beta_features.spruce_waterfall_enabled": boolean };
```